workers/analyzer/src/main/kotlin/analyzer/AnalyzerWorker.kt

```diff
@@ -100,7 +102,11 @@ internal class AnalyzerWorker(
             ortRunService.storeAnalyzerRun(analyzerRun.mapToModel(jobId))
         }
 
-        RunResult.Success
+        if (analyzerRun.result.issues.values.flatten().any { it.severity >= Severity.WARNING }) {
```

Comment:
Call `analyzerRun.result.getAllIssues()` instead, the current call does not get issues from the dependency tree.
This should appply issue resolutions to ignore resolved issues, but we can do that in a follow-up PR, because I think it requires some more changes and would make this PR too large.